zpeclient.go

```diff
@@ -410,3 +412,11 @@ func (c *Cache) updateCacheStatus(timestamp string) error {
 	c.cacheStatus = CacheStale
 	return nil
 }
+
+// SetCacheEnabledStatus - sets cacheEnabled value
+// when cacheEnabled is set to true, cache will be used in the request flow.
+func (c *Cache) SetCacheEnabledStatus(status  bool) {
```

Comment:
could we call the `status` arg as `enabled` to avoid confusion with `cacheStatus` [here](https://github.com/yahoo/k8s-athenz-webhook/pull/26/files#diff-573469fad3a1d667ff08570ca2a40cc382cf8390d706d4fee74802319068a830R202)?

Reply:
Updated the variable.